packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.ts

```diff
@@ -84,56 +89,107 @@ export class RipgrepSearchInWorkspaceServer implements SearchInWorkspaceServer {
     @inject(RgPath)
     protected readonly rgPath: string;
 
+    private static readonly GLOB_SUB_DIRECTORY_PATTERN = '**/';
+    private static readonly GLOB_COMMAND_ARGUMENT = '--glob=';
+
     constructor(
         @inject(ILogger) protected readonly logger: ILogger,
         @inject(RawProcessFactory) protected readonly rawProcessFactory: RawProcessFactory,
     ) { }
 
+    private get globSubDirectory(): string {
+        return RipgrepSearchInWorkspaceServer.GLOB_SUB_DIRECTORY_PATTERN;
+    }
+
+    private get globCommandArgument(): string {
+        return RipgrepSearchInWorkspaceServer.GLOB_COMMAND_ARGUMENT;
+    }
```

Comment:
The following can be removed, the `GLOB_SUB_DIRECTORY_PATTERN` and `GLOB_COMMAND_ARGUMENT` are already accessible statically.

Reply:
I added the getters as I find it quite undreadable to use the long class name directly, I am tempted to just use a const directly in the function to avoid the long names  and getters, any other ideas ?

Reply:
I don't think these static variables are terribly useful, we can create `const` directly where they are used instead if you prefer.

Reply:
I remove the getters, hopefully we find a better way in the future, hopefully new versions of typesscript allow to specify the variable directly without the class name while in the same class.